client/stack-editor/lib/index.coffee

```diff
@@ -64,6 +62,15 @@ module.exports = class StackEditorAppController extends AppController
           @removeEditor stackTemplate._id
           kd.singletons.router.handleRoute '/IDE'
 
+        isClonedTemplate stackTemplate, (originalTemplate) =>
+          if originalTemplate
+            editor.addClonedFrom originalTemplate
+            originalTemplate.on 'update', (update) ->
+              if stackTemplate.template.sum isnt originalTemplate.template.sum
+                return editor.addCloneUpdateView originalTemplate
+            if stackTemplate.template.sum isnt originalTemplate.template.sum
```

Comment:
It looks weird but when stack-editor view is open for member, and when admin update the original stack template, I need to add the update event here.